crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -266,6 +266,8 @@ async fn symbolicate_js_frame(
     frame.lineno = token.line().saturating_add(1);
     frame.colno = Some(token.column().saturating_add(1));
 
+    frame.data.symbolicated = true;
+
```

Comment:
Question: is this place, immediately after setting the line and column number, the point where we can say that the frame has been successfully symbolicated?

Reply:
I would say yes. Maybe even earlier. If we have a matching token, it means we symbolicated *something*. Even though the token could be wrong due to using a non-matching sourcemap, for whatever reason.

Reply:
Ok, I moved the statement and added a comment to that effect. Semantically it shouldn't make a difference because the function can't return between these points, but I think it's clearer this way.